README.md

```diff
@@ -14,55 +23,53 @@ In this modified Cell Painting, there are five channels:
 
 ![Composite_Figure.png](example_figs/Composite_Figure.png)
 
-## Plate maps
+## Data
 
-We applied this modified Cell Painting assay using the following plate design for the first two plates:
+For training a logistic regression classifier, we extracted single-cell morphology profiles from CFs of different patients with the same heart failure type and patients with non-failing hearts.
+We label wells from each patient as either "Failing" (failed heart) or "Healthy" (non-failing heart).
+There was no treatment or perturbation applied to any of the wells in this plate.
 
-- **localhost220512140003_KK22-05-198**
+- **localhost231120090001**
+
+![localhost231120090001_platemap_figure.png](./metadata/platemap_figures/localhost231120090001_platemap_figure.png)
 
-![localhost220512140003_KK22-05-198_platemap_figure.png](./metadata/platemap_figures/localhost220512140003_KK22-05-198_platemap_figure.png)
+---
 
-- **localhost220513100001_KK22-05-198_FactinAdjusted**
+We utilized a secondary plate to assess the generalizability of the model.
+This plate contains two different hearts, one from a non-failing patient and the other from a failing patient.
+Wells are labelled as either "Failing" (failing heart) or "Healthy" (non-failing heart).
+There are three different treatments:
 
-![localhost220513100001_KK22-05-198_FactinAdjusted_platemap_figure.png](./metadata/platemap_figures/localhost220513100001_KK22-05-198_FactinAdjusted_platemap_figure.png)
+- DMSO (control)
+- drug_x
+- TGRFi (TGFB inhibitor)
 
-For the third plate, we are using the following plate design:
+We applied the model to this place to assess that the model is able to accurately predict the control cells and how the model predictions change with the treatments.
```

Comment:
```suggestion
We applied the model to this dataset to evaluate its accuracy in predicting control cells and to observe how its predictions change in response to different treatments.
```

Feel free to reject if my edits generates a different message you're trying to convey.

Reply:
This work really well, updated!